game/lib/global_leaderboard/global_leaderboard_widget.dart

```diff
@@ -83,132 +93,129 @@ class _GlobalLeaderboardWidgetState extends State<GlobalLeaderboardWidget> {
+                      width: 360.0,
+                      height: 446.0,
+                      decoration: BoxDecoration(
+                        color: Color.fromRGBO(255, 170, 91, 0.15),
+                        borderRadius: BorderRadius.only(
+                          topLeft: Radius.circular(10.0),
+                          topRight: Radius.circular(10.0),
                         ),
-                        child: Container(
-                          width: 283.05,
-                          height: 432.0,
-                          child: ListView(
-                            shrinkWrap: true,
-                            scrollDirection: Axis.vertical,
-                            children: [
-                              for (LeaderDto user in list.skip(3))
-                                Padding(
-                                  padding: const EdgeInsets.only(
-                                      left: 30.95, right: 30.95, top: 16.0),
-                                  child: Container(
-                                    decoration: BoxDecoration(
-                                      borderRadius: BorderRadius.only(
-                                        topLeft: Radius.circular(10.0),
-                                        topRight: Radius.circular(10.0),
-                                        bottomLeft: Radius.circular(10.0),
-                                        bottomRight: Radius.circular(10.0),
-                                      ),
-                                      color: Colors.white,
-                                      boxShadow: [
-                                        BoxShadow(
-                                          color: Color(0x40000000),
-                                          offset:
-                                              Offset(0.0, 1.7472529411315918),
-                                          blurRadius: 6.989011764526367,
-                                        ),
-                                      ],
-                                    ),
-                                    child: leaderBoardCell(
-                                      context,
-                                      user.username,
-                                      position++,
-                                      user.score,
-                                      user.userId == myUserModel.userData?.id,
+                      ),
+                      child: Container(
+                        width: 283.05,
+                        height: 432.0,
+                        child: ListView(
+                          shrinkWrap: true,
+                          scrollDirection: Axis.vertical,
+                          children: [
+                            for (LeaderDto user in list.skip(3))
+                              Padding(
+                                padding: const EdgeInsets.only(
+                                    left: 30.95, right: 30.95, top: 16.0),
```

Comment:
I'm guessing you are adding this padding to center the container? You should wrap your Container child in a Align widget to center it instead. This is why the leaderboard is not centered when we tested it on our deployed iOS devices!